### PR TITLE
modify unified_focal_loss.py

### LINKED ISSUE #8603: Add sigmoid/softmax interface for `AsymmetricUnifiedFocalLoss`
**Is your feature request related to a problem? Please describe.**
It seems that `AsymmetricUnifiedFocalLoss` has not yet been updated to the latest interface like `FocalLoss`. It is a general loss function and we should make it easier for everyone to use. We can also allow multiple classes segmentation since it only supports binary one accordingly

**Describe the solution you'd like**
Follow the interface of `FocalLoss`. Also, we don't need to reinvent the wheel. I am sure we can use the available loss function to replace the [implementation](https://github.com/Project-MONAI/MONAI/blob/dev/monai/losses/unified_focal_loss.py#L189).

**Describe alternatives you've considered**
No

**Additional context**
No

### PR DESCRIPTION
**Pull request by ytl0623**
Fixes #8603

### Description

Asymmetric Unified Focal Loss with sigmoid/softmax interface and multiclass support.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by ericspod** [comment]
Hi @ytl0623, appreciate the contribution with these losses. As we're adding new arguments such as `include_background`, we should have tests for these being either True or False so that we're sure the classes work in both cases. These can be quite simple tests but should go with the existing tests for these classes. Thanks!
